packages/executor/.env.example

```diff
@@ -1,6 +1,8 @@
 CHUGSPLASH_EXECUTOR__PRIVATE_KEY=<private key to deploy with>
-CHUGSPLASH_EXECUTOR__NETWORK=<network to monitor, http://host.docker.internal:8545 for local node>
+CHUGSPLASH_EXECUTOR__URL=<url of the network to monitor, http://host.docker.internal:8545 for local node>
+CHUGSPLASH_EXECUTOR__NETWORK=<name of the network to monitor, i.e 'hardhat' or 'optimism-goerli'>
```

Comment:
What's the difference between this and `HARDHAT_NETWORK`?

Reply:
`CHUGSPLASH_EXECUTOR__NETWORK` sets the `network` option in the executor. The network option is used to fetch the etherscan verification endpoint.